packages/@guidesmiths/cuckoojs-schematics/src/helm/helm-strategies/tls-cert/none.tlsCert.strategy.ts

```diff
@@ -0,0 +1,8 @@
+import {TlsCertStrategy} from './tlsCert.strategy';
+
+export class NoneTlsCertStrategy extends TlsCertStrategy {
+	constructor() {
+		super(false);
+		this.templatePath.push('none');
```

Comment:
Is it possible to declare an enum in ingress.strategy.factory.ts with the `none` and `certManager` values to avoid hardcoded values?